PocketKit/Sources/PocketKit/Main/MainView.swift

```diff
@@ -83,12 +83,12 @@ public struct MainView: View {
             }
         }
         .zIndex(-1)
-        .onChange(of: featureFlag, initial: false) {
-            guard let swiftuiFeatureFlag = featureFlag.first else {
-                return
-            }
-            assigned = swiftuiFeatureFlag.assigned
-        }
+//        .onChange(of: featureFlag, initial: false) {
```

Comment:
@Gio2018 FYI

Reply:
oh yea thanks for finding it! Just reverted, serves to see the SwiftUI Home when logged out, since it's feature flagged